### ISSUE TITLE
Go to implementation (for interfaces)

### ISSUE THREAD

**Issue by joshtynjala** [prominic/Moonshine-IDE#299]
Expose the results of the `vscode.executeImplementationProvider` command.

https://code.visualstudio.com/docs/editor/editingevolved#_go-to-implementation

Will be supported in version 0.13 of the language server.

**Comment by JoelProminic** [comment]
I did some tests on macOS with this feature while reviewing the Changelog for the 2.4.1/2.5.0 release.

I found that Edit > Go To Implementation was not enabled at all for Java Gradle projects.  Which project types are supported currently?

I created an example Flex Desktop project to test this.  However, the popup window does not show any entries in my test.  Here is my project:  

[2019_11_08__TestFlexInterfaces.zip](https://github.com/prominic/Moonshine-IDE/files/3825730/2019_11_08__TestFlexInterfaces.zip)

My procedure:
1. Open project in latest Moonshine build
1. Open src/ExampleInterface.as
1. Put the cursor on the `test()` method name, and click Edit > Go To Implementation
1. BUG:  The opened window does not have any entries.  I expect this to show ExampleImplementation.as and ExampleImplementation2.as
1. Open src/ExampleImplementation2.as
1. Put the cursor on the `test()` method name, and click Edit > Go To Implementation
1. BUG:  The list is again empty

![image](https://user-images.githubusercontent.com/24532731/68500742-0b4d9980-0222-11ea-9d20-0e16e72acf02.png)

Let me know if I am misunderstanding how this feature is supposed to work - this is not a feature I have used in other SDKs.

**Comment by joshtynjala** [comment]
> Put the cursor on the test() method name, and click Edit > Go To Implementation

Place the cursor on the name of an interface instead. It should show a list of classes that implement the interface.

I could potentially add support for interface methods with this command too, but it is not possible at this time. It would need to be implemented in a future update to the AS3 & MXML language server.

> I found that Edit > Go To Implementation was not enabled at all for Java Gradle projects. Which project types are supported currently?

I enabled this command for AS3 & MXML projects because I know that it is supported in the AS3 & MXML language server.

It's been long enough that I don't recall whether I checked or not, but it's possible that the Java language server simply does not support this particular command. I can look into it again.
